### PR TITLE
Create ci-scan.yml

### PR DESCRIPTION
**Pull request by vineetmimrot**

### CODE REVIEW

File: .azure-pipelines/ci-scan.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+pool:
```
Comment (bishal-pdMSFT) [inline .azure-pipelines/ci-scan.yml]: how will this yml get consumed? I don't see it being consumed in azure-pipelines.yml?

File: .azure-pipelines/ci-scan.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  name: Hosted VS2017
```
Comment (bishal-pdMSFT) [inline .azure-pipelines/ci-scan.yml]: can it run on hosted ubuntu? build/package pipelines runs on ubuntu

Comment (vineetmimrot) [reply .azure-pipelines/ci-scan.yml]: I don't think that will matter, as credscan & TSLint just scans the code and doesn't build it.

File: .azure-pipelines/ci-scan.yml (hunk 1 above)
Comment (vineetmimrot) [reply .azure-pipelines/ci-scan.yml]: There will be separate ci for this
https://dev.azure.com/mseng/AzureDevOps/_build?definitionId=10036&_a=summary

**Pull request closed without merging**